### DISCUSSION TITLE
Keyboard Select

### DISCUSSION THREAD

**Discussion by fightforvain** [ahrm/sioyek#329] [Q&A]
How to use keyboard selection to select only one word？ I try to input same symbol such as <word tag><space><word tag>, but it didn't work. I also try to input <current word tag><space><next word tag>, but  it selected some part of next word.

**Comment by fightforvain** [comment]
Thanks a lot for you reply! I get it. I found that sometimes it works well, but the keyboard can not select exactly&nbsp;for some complex words. Such as the word literature-curated lose the letter "l" when I use the keyboard select.&nbsp; &nbsp;








------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ahrm/sioyek"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年7月18日(星期一) 下午4:20
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ahrm/sioyek] Keyboard Select (Discussion #329)



 You need to enter the space-separated symbols for the beginning and the end for example: am gh.

**Reply by ahrm** [reply]
Yes, I realized I did not understood what you meant correctly and deleted my reply. I guess you received the email anyway.
